vpr/src/pack/cluster_router.cpp

```diff
@@ -478,7 +479,13 @@ bool try_intra_lb_route(t_lb_router_data *router_data,
 			//
 			// All nets from each pb_type must not have a mode conflict between
 			// the nets.
-			is_impossible = check_for_route_conflicts(lb_nets, router_data);
+			bool conflicts_found = check_for_route_conflicts(lb_nets, router_data);
+			if (conflicts_found) {
+				// The nets are rerouted in a recursive fashion until no more possible
+				// modes are applicable.
+				// TODO: add stop condition not to end up in a possible infinite recursive loop!
+				is_impossible = !try_intra_lb_route(router_data, verbosity);
```

Comment:
This is also deeping the stack, probably not a good plan.

Reply:
Yes, I agree, I am not a fan of recursive calls. Now that I know exactly what the issue is I will think of a non-recursive solution

Reply:
Done, even though I believe it should be further refined